src/internal/minikubetestenv/deploy.go

```diff
@@ -494,6 +495,7 @@ func putRelease(t testing.TB, ctx context.Context, namespace string, kubeClient
 	}
 	if opts.EnterpriseServer {
 		helmOpts = union(helmOpts, withEnterpriseServer(version, pachAddress.Host))
+		helmOpts = union(helmOpts, withMinio())
```

Comment:
I want to check with Alon when he comes back to see if this is okay. We currently need this to set 'deployTarget' when running in enterprise server mode. The deployTarget gets set in 'withMinio()'